src/fides/api/schemas/redis_cache.py

```diff
@@ -44,8 +44,14 @@ def validate_fides_user_device_id(cls, v: Optional[str]) -> Optional[str]:
         return v
 
 
+CustomPrivacyRequestFieldValue = Union[
+    Union[StrictInt, StrictStr], List[Union[StrictInt, StrictStr]]
+]
+
+
 class CustomPrivacyRequestField(FidesSchema):
     """Schema for custom privacy request fields."""
 
     label: str
-    value: Any
+    # use StrictInt and StrictStr to avoid type coercion and maintain the original types
+    value: CustomPrivacyRequestFieldValue
```

Comment:
Would there be any issues with serializing previously saved data that was less strict?

Reply:
We should be good here. I added the `test_integration_custom_privacy_request_fields.py` tests to `main` and a request with dict values failed very early at the caching step, before we have a chance to persist anything to the database.

Reply:
but what if there are already custom fields saved in a customer's implementation? is that possible?

Reply:
dicussed in person - before there was stricter validation, lists and dict values would fail earlier because other types couldn't be persisted to the db, so it sounds like only ints and strings could be saved in the first place